lib/components/artworks/artwork.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+import Relay from 'react-relay';
+import React from 'react-native';
+const { Text, View, StyleSheet } = React;
+
+import ImageView from '../opaque_image_view';
+import colors from '../../../data/colors';
+
+export default class Artwork extends React.Component {
+  render() {
+    const artwork = this.props.artwork;
+    const optionals = [];
+    // TODO partner ?
+    if (artwork.sale_message) {
+      optionals.push(<Text key="sale_message">{this.props.artwork.sale_message}</Text>);
+    }
+    return (
+      <View>
+        <ImageView style={styles.image} aspectRatio={artwork.image.aspect_ratio} imageURL={artwork.image.url} />
+        <Text>{artwork.artist.name}</Text>
+        <Text>{artwork.title}</Text>
+        <Text>{artwork.partner.name}</Text>
```

Comment:
Ah, good catch, this component I made based on a generalized artwork component design, it should indeed be updated to match the spec for the artist page.